packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -32,7 +33,7 @@
     "webpack": "5.68.0"
   },
   "scripts": {
-    "compile": "webpack",
+    "compile": "tsc -p ./ && webpack",
```

Comment:
Updated to be the same as our other webpack builds.  This was weird previously in that it was using webpack to do the tsc compile.  Now all of our webpack builds are very similar.